app/components/markdown/markdown.js

```diff
@@ -223,6 +215,7 @@ export default class Markdown extends PureComponent {
                 isSearchResult={this.props.isSearchResult}
                 mentionName={mentionName}
                 onPostPress={this.props.onPostPress}
+                mentionKeys={this.props.mentionKeys}
```

Comment:
based on my question in `at_mention/index.js` this should not be needed.

Reply:
Yes it is needed. We have to pass it down since `markdown.js` needs the mentionKeys.